### DISCUSSION TITLE
Multi-Tenancy get the latest tenant

### DISCUSSION THREAD

**Discussion by fagnerdireito** [filamentphp/filament#7517] [General]
Hello, I have a question regarding Multi-Tenant. 

I'm trying to get the default tenant, latest tenant, but I'm not sure if I'm doing it correctly. 

I'm using these functions:

`  public function getDefaultTenant(): ?Model
    {
        return $this->latestTeam;
    }
`

`public function latestTeam(): BelongsTo
    {
        return $this->belongsTo(Team::class, 'latest_team_id');
    }`

but I don't see the ?Model attribute, so I changed it to the ?Team, and return this error:

> Declaration of App\Models\User::getDefaultTenant(): ?App\Models\Team must be compatible with Filament\Models\Contracts\HasDefaultTenant::getDefaultTenant(Filament\Panel $panel): ?Illuminate\Database\Eloquent\Model

i try change set ?Team model because I believe it refers to my teams model. However, the system returns an error saying that App\Models\Team is not compatible with getDefaultTenant. 

Can someone help me?

**Comment by pboivin** [comment]
The return type has to follow what's in the HasDefaultTenant interface.

**Reply by fagnerdireito** [reply]
how to try yhis?

**Reply by pboivin** [reply]
Have you tried this?
```php
use Illuminate\Database\Eloquent\Model;

public function getDefaultTenant(): ?Model
{
    // ...
}
```

**Comment by hamstag** [comment]
try this with Panel $panel param
```
use Filament\Panel;

public function getDefaultTenant(Panel $panel): ?Model
{
    return $this->latestTeam;
}
```
